### ISSUE TITLE
Didn't find softmax in NN models

### ISSUE THREAD

**Issue by honshj** [jeffreysijuntan/CryptGPU#3]
Hi, I didn't see the softmax in all NN models, does it mean we don't need it in private training?  Thanks!

**Comment by jeffreysijuntan** [comment]
We do need softmax during training, but we do not call softmax explicitly. Softmax is invoked implicitly when computing the cross entropy loss (https://github.com/jeffreysijuntan/CryptGPU/blob/master/crypten/gradients.py#L1273)
